schemas/stsci.edu/asdf/0.1.0/asdf-schema.yaml

```diff
@@ -0,0 +1,46 @@
+%YAML 1.1
+---
+$schema: "http://json-schema.org/draft-04/schema"
+id: "http://stsci.edu/schemas/asdf/0.1.0/asdf-schema"
+title:
+  ASDF schema
+description: |
+  A metaschema extending YAML Schema and JSON Schema to add support
+  for some ASDF-specific checks, related to nd-arrays.
```

Comment:
This allows us to write schemas that validate the type or dimensions of an array.  Presently, all we can say is that it *is* an array.  And we can say what keys an array description should have.  But we need to extend the metaschema in order to add new validation types.  The use case for this is in the JWST pipeline, where we want to say a particular file contains a 2D image or a 3D ramp or 4D cube etc.  But this is already being used internally (in this PR) where you see it can declare that the data type of a mask is boolean.